wallet/src/de/schildbach/wallet/ui/LockScreenActivity.kt

```diff
@@ -512,5 +515,19 @@ open class LockScreenActivity : SecureActivity() {
         }
         bottomSheetDialog?.dismiss()
         dialogFragment?.dismiss()
+
+        dismissOffsetDialogs(supportFragmentManager)
+    }
+
+    private fun dismissOffsetDialogs(fragmentManager: FragmentManager) {
```

Comment:
There is another problem with dismissing dialogs currently - if the user unlocked the app and then opened a dialog, `activatingLockScreen` can be triggered after initialization, which will lead to that dialog being dismissed immediately.

I have added an alternative method for the `OffsetDialogFragment` exclusively to avoid side effects, but we should consider dismissing all dialogs this way.